main/libmicrohttpd/APKBUILD

```diff
@@ -31,7 +31,13 @@ build() {
 
 check() {
 	cd "$builddir"
-	make check
+	make check || true
+	# Travis-CI log check
+	msg "cat test_options.log"
+	cat src/microhttpd/test_options.log
+	msg "cat test-suite.log"
+	cat src/microhttpd/test-suite.log
+	false
 }
```

Comment:
Please revert changes to the `check` function as well.

Reply:
Reverted. I have added that code to debug Travis-CI build system.